### ISSUE TITLE
[Accessibility]A11y_VSCode_ShowOpenedEditors_ScreenReader:NVDA does not read the search results found after typing any command in editor.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52325]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10
**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA 2018.2,JAWS18.0.5038
**Repro Steps:**
Launch VS Code while JAWS/NVDA is ON.

1.Navigate   to welcome screen and beside opened editors pane at top(in middle   pane)->Select More(…)->Select "Show Opened Editors" from   dropdown.
2.Navigate to editor and search for any commands.
**Actual:**
JAWS /NVDAdoes not read the "No results found....." content which tells user the search result did not produce any valid result.
**Expected:**
JAWS /NVDA should read the "No results found...." content as soon as search is performed by user and no outputs are found.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
**User Impact:**
Screenreader users will not know about the empty search content and other meaningful information.When "No results...." content is read ,the users will then and there come to know about the search results output.
**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes/No
**Attachment for Reference:**
[usability.pptx](https://github.com/Microsoft/vscode/files/2115072/usability.pptx)

**Comment by roblourens** [comment]
I don't understand the scenario, where are you finding "Show more"? Is step 1 necessary to repro this?

And in step 2, what do you mean "search for any commands"? Do you mean use the command palette (ctrl+shift+P)? Are you just using quickopen (ctrl+P), there the message is "No results found".

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52542 and https://github.com/Microsoft/vscode/issues/52383
